netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -170,6 +170,9 @@ public final NettyChannelBuilder eventLoopGroup(@Nullable EventLoopGroup eventLo
    * GrpcSslContexts}, but options could have been overridden.
    */
   public final NettyChannelBuilder sslContext(SslContext sslContext) {
+    if (sslContext != null && sslContext.isServer()) {
```

Comment:
Thanks. I saw two Precondition classes in the code base. One is in the package com.google.api.client.util, the other is in com.google.common.base. Which am I supposed to use?